shared/scss/views/_privacy-practices.scss

```diff
@@ -0,0 +1,125 @@
+.privacy-practices {
+    min-height: 100%;
+    box-sizing: border-box;
+}
+
+.privacy-practices__overview,
+.privacy-practices__explainer,
+.privacy-practices__attrib {
+    text-align: center;
+}
+
+// the domain duplicate the same styles
+// as the ones in the main view, but they aren't reusable
+// because of some overriding rules for the subviews
+// TODO DRY this up
+.privacy-practices__overview__domain {
+    line-height: 1.4em;
+    height: 1.2em;
+    display: block;
+    max-width: $popup__width - 125;
+    font-weight: bold;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    margin-top: 7.5px;
+    text-align: center;
+    width: 100%;
+    max-width: 100%;
+    font-size: 17px;
+}
+
+.privacy-practices__overview__subtitle {
+    padding-top: 7.5px;
+    @include uppercase_label();
+}
+
+.privacy-practices__overview__icon {
+    width: 48px;
+    height: 65px;
+    margin: 0 auto;
+    background-repeat: no-repeat;
+}
+
+// TODO dry these up also
+.privacy-practices__overview__icon--good {
```

Comment:
@andrey-p Oh now I know what you meant earlier. Yes, this is a good usecase for doing something like [this](https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/1/files#diff-2dff5e5386b765d8477d1bbb024acc20)